server/www/templates/partials/user-menu.html

```diff
@@ -1,26 +0,0 @@
-<ul class="account-menu">
-<!-- user account menu -->
-    <li>
-        <a ng-href="{{baseUrl + view.name }}">
-            <img class="avatar" src="http://www.gravatar.com/avatar/{{ currentUser.gravatar || '00000000000000000000000000000000' }}?d=retro" alt="{{ currentUser.realname }}"/>
-            <span class="label">{{currentUser.realname || 'Aaliyah Williams'}}</span>
-        </a>
-    </li>
-    <li>
-        <a ng-href="{{baseUrl + view.name }}">
-            <svg class="iconic">
-                <use xlink:href="../../img/iconic-sprite.svg#grid-three-up"></use>
-            </svg>
-            <span class="label">Collections</span>
-        </a>
-    </li>
-    <li>
-        <a ng-href="{{baseUrl + view.name }}">
```

Comment:
@middle8media try to avoid href's that will just link to the current page. If they're no supposed to trigger a URL change remove the `href` entirely.